resources/issues.md

```diff
@@ -33,3 +33,11 @@ Why? People often ask to be assigned and never come back, which discourages the
 We will accept and merge the first PR that fixes correctly and implements the issue following the contribution guidelines of the corresponding repository.
 
 We are looking forward to your contributions. No need to wait for an assignment to start! 👍
+
+## Issue priority feedback policy based on tier level
+
+Hey \@username.
+I agree on implementing this! 
+
+We might not have enough time to work on it in the following weeks.
+If someone wants to contribute and implement it, it will be released faster! PRs are always welcomed!
```

Comment:
```suggestion
## Issue priority feedback policy based on tier level

Hey \@username.
I agree with implementing this! 

We might not have enough time to work on it in the following weeks. Also, I will add some details to this issue to make the implementation easier.
If someone wants to contribute and implement it, it will be released faster! PRs are always welcome!
```